pymongo/operations.py

```diff
@@ -374,7 +375,7 @@ class UpdateMany(_UpdateOp):
     def __init__(
         self,
         filter: Mapping[str, Any],
-        update: Union[Mapping[str, Any], _Pipeline],
+        update: Union[_DocumentType, RawBSONDocument, _Pipeline],
```

Comment:
Woah, the update doc shouldn't accept the _DocumentType. The update is a mapping like this `{"$inc": ...}`, not a full doucment.